client-js/app/elements/labrad-registry.html

```diff
@@ -142,15 +140,15 @@
           <iron-icon icon="folder-open"></iron-icon>
           new dir
         </paper-button>
-        <paper-button on-click="copyClicked" disabled="{{!selected}}">
+        <paper-button on-click="copyClicked" disabled="{{selected}}">
```

Comment:
Seems strange to change the sense of `selected` here. Shouldn't these buttons be disabled only if nothing is selected?